config/action.d/nftables-common.conf

```diff
@@ -0,0 +1,96 @@
+# Fail2Ban configuration file
+#
+# Author: Daniel Black
+# Author: Cyril Jaquier
+# Modified: Yaroslav O. Halchenko <debian@onerussian.com>
+# 			made active on all ports from original iptables.conf
+# Modified: Alexander Belykh <albel727@ngs.ru>
+#                       adapted for nftables
+#
+# This is a included configuration file and includes the definitions for the nftables
+# used in all nftables based actions by default.
+#
+# The user can override the defaults in nftables-common.local
+
+[INCLUDES]
+
+after = nftables-common.local
+
+[Definition]
+
+# Option:  nftables_mode
+# Notes.:  additional expressions for nftables filter rule
+# Values:  nftables expressions
+#
+nftables_mode = <protocol> dport { <port> }
+
+# Option:  actionstart
+# Notes.:  command executed once at the start of Fail2Ban.
+# Values:  CMD
+#
+actionstart = <nftables> add set filter f2b-<name> { type ipv4_addr\; }
```

Comment:
I am yet to learn so much about this new beast... but FWIW in Debian default configuration either doesn't have some generic "filter" but has "inet filter" (not yet even sure if inet used as a family or just a part of two word identifier here, or both):
```
> head /etc/nftables.conf  # that is the default configuration
#!/usr/sbin/nft -f

flush ruleset

table inet filter {
	chain input {
...
```
```
> nft list table inet filter
table inet filter {
	chain input {
		type filter hook input priority 0; policy accept;
		iif lo accept 
		ct state established,related accept 
		ip6 nexthdr ipv6-icmp icmpv6 type { nd-router-advert, nd-neighbor-solicit, nd-neighbor-advert} accept 
		counter packets 11 bytes 1314 drop 
	}
}
```
so I need "nft add set inet filter f2b-test..."

but if that is the truth of life in nft world, we might need an additional parameter identifying that `filter` to manipulate